packages/@ama-sdk/schematics/schematics/ng-update/index.ts

```diff
@@ -2,14 +2,15 @@
 /* eslint-disable camelcase */
 
 import { Rule, Tree } from '@angular-devkit/schematics';
-import { updateV10_0 as tsUpdateV10_0 } from './typescript';
+import { updateV10_0 as tsUpdateV10_0, updateV10_3 as tsUpdateV10_3 } from './typescript';
+import { globInTree } from '@o3r/schematics';
 
 /**
  * Determine if the script is run in a Typescript SDK
  * @param tree
  */
 const isTypescriptSdk = (tree: Tree) => {
-  return tree.exists('/tsconfig.json');
+  return globInTree(tree, ['/tsconfig.*json']);
```

Comment:
```suggestion
  return globInTree(tree, ['/tsconfig.*.json']);
```
(same thing but the `*json` is not usual)